loopslab.js

```diff
@@ -0,0 +1,64 @@
+// Number 1a Write a while loop and a for loop that takes the variable "num" and logs all the numbers,
+//descending, between "num" and 1.
+//let num = 15;
+//while (num >= 1) {
+ //console.log (num);
+ //num --
+//}
+
+
+// //1b
+//for (let num = 15; num >= 1; num --) {
+//console.log (num);
+//}
+
+//
+// Number 2 Write a while loop and a for loop that takes the variable "max",
+//and iterates over all numbers from 0 to "max". For each iteration,
+//it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
+//
+//
+//let max = 20;
+//while (max >= 0){
+//if (max % 2 !== 0)
+  //console.log(max + " is odd");
+//if (max %2 === 0)
+//console.log(max + "is even")
+//max --
+//}
+//
+//
+// // 2a (Not complete)
+
+// for (let max = 20); max >= 0; max --) {
+//   console.log (num + "even")
+
+//Number 3:
+
+//Write a while loop and a for loop that takes the variable "x"
+//and iterates over all numbers from 0 to "x".
+//For each iteration of the loop, it will multiply the number of "x" by 9 and log the result (e.g. "2 * 9 = 18").
+//Bonus think of another way to solve it.
+//
+
+//
+//Number 4: Write code that prints/logs all the numbers that end in a 5 from 1 to 100, exclusive.
+//My answer does not end in 5
+
+  //let num = 5
+  //while (num <= 100) {
+    //console.log(num);
+    //num += 10
+  //}
+//
+// Number 5 :Without running/executing your code, how many times will the loop below run? Explain why.
+//
+//   let i = 5;
+//
+// while (i > 3) {
+//     i += 1
+// }
+// It will run an infinite number of times
```

Comment:
Correct. It will run indefinitely since variable i will always be bigger than 3.